styles/overrides/_popover.scss

```diff
@@ -1,4 +1,6 @@
 .popover.bottom {
```

Comment:
Remind me, is the `.bottom` class just for the bottom of the popover (e.g. the content) or is is when the popover is positioned at the bottom of the element you're hovering/clicking on?

Reply:
Hmm probably shouldn't use .bottom as its very specific. Should I add an extra class you think and use that?